src/images/elastic_search.rs

```diff
@@ -0,0 +1,56 @@
+use std::collections::HashMap;
+
+use crate::{core::WaitFor, Image};
+
+const NAME: &str = "docker.elastic.co/elasticsearch/elasticsearch";
+const TAG: &str = "7.16.1";
+
+#[derive(Debug)]
+pub struct ElasticSearch {
+    env_vars: HashMap<String, String>,
+    tag: String,
+}
+
+impl Default for ElasticSearch {
+    fn default() -> Self {
+        let mut env_vars = HashMap::new();
+        env_vars.insert("discovery.type".to_owned(), "single-node".to_owned());
+        ElasticSearch {
+            env_vars,
+            tag: TAG.to_owned(),
+        }
+    }
+}
+
+impl ElasticSearch {
+    pub fn with_tag(self, tag: String) -> Self {
+        Self { tag, ..self }
+    }
```

Comment:
Same here, the tag can be overridden via `RunnableImage`.

Reply:
Done